stratum/hal/lib/barefoot/bf_chassis_manager.h

```diff
@@ -206,6 +209,19 @@ class BFChassisManager {
   std::map<uint64, std::map<uint32, PortKey>>
       node_id_to_port_id_to_singleton_port_key_ GUARDED_BY(chassis_lock);
 
+  // Map from node ID to another map from (SDN) port ID to SDK port ID.
+  // SDN port IDs are used in Stratum and by callers to P4Runtime and gNMI,
+  // and SDK port IDs are used in calls to the BF SDK. This map is updated
+  // as part of each config push.
+  std::map<uint64, std::map<uint32, uint32>>
+      node_id_to_port_id_to_sdk_port_id_ GUARDED_BY(chassis_lock);
+
+  // Map from node ID to another map from SDK port ID to (SDN) port ID.
+  // This contains the inverse mapping of: node_id_to_port_id_to_sdk_port_id_
+  // This map is updated as part of each config push.
+  std::map<uint64, std::map<uint32, uint32>>
+      node_id_to_sdk_port_id_to_port_id_ GUARDED_BY(chassis_lock);
```

Comment:
Need to add both to the constructor's [member initializer list](https://en.cppreference.com/w/cpp/language/constructor)